src/main/steps/common/confirm-contact-details/address/lookup/content.ts

```diff
@@ -0,0 +1,61 @@
+import { TranslationFn } from '../../../../../app/controller/GetController';
+import { FormContent, FormFields, FormFieldsFn } from '../../../../../app/form/Form';
+import { ResourceReader } from '../../../../../modules/resourcereader/ResourceReader';
+import {
```

Comment:
Seems duplicate file to one present under /applicant directory. can we remove one and keep this file under /common directory?

Reply:
Will complete refactoring as part of follow up ticket - other refactoring required throughout the different journeys